packages/components/src/Token/Form/Create.tsx

```diff
@@ -298,18 +304,27 @@ const TokenFormCreate: FC<Props> = ({
         <InputGroup>
           <NumberInput
             clampValueOnBlur={false}
-            min={0}
-            max={maxRoyalties}
-            step={0.01}
             allowMouseWheel
             w="full"
+            step={0.5}
```

Comment:
Let's keep the existing step:
```suggestion
            step={0.01}
```

Reply:
Updated